Source/DataSources/KmlDataSource.js

```diff
@@ -967,6 +969,40 @@ define([
         entity.polyline.positions = new PositionPropertyArray([entityPosition, surfacePosition]);
     }
 
+    function heightReferenceFromAltitudeMode(altitudeMode, gxAltitudeMode) {
+        if (!defined(altitudeMode) && !defined(gxAltitudeMode) || altitudeMode === 'clampToGround') {
+            return HeightReference.CLAMP_TO_GROUND;
+        }
+
+        if (altitudeMode === 'relativeToGround') {
+            return HeightReference.RELATIVE_TO_GROUND;
+        }
+
+        if (altitudeMode === 'absolute') {
+            window.console.log('KML - <kml:altitudeMode>:absolute is currently not supported, using <kml:altitudeMode>:relativeToGround.');
```

Comment:
Never use `window.` because it will break JavaScript implementations that don't have window (like Node).  Instead just use `console.log` directly, since `window` is the global object and the result is exactly the same.

However, would it be better for these to be `oneTimeWarning`?